helm_charts_updater/git.py

```
@@ -58,37 +145,72 @@ def push_changes(self, chart_version, app_name: str, version: str, old_version:
             self.local_repo.git.pull(
-                origin, self.local_repo.active_branch.name, strategy_option="ours", rebase=True
+                origin,
+                self.local_repo.active_branch.name,
+                strategy_option="ours",
+                rebase=True,
             )
         except GitCommandError as e:
-            logging.error("Error while pulling with rebase: %s", e)
-            raise GitCommandError
+            sanitized_error = _sanitize_url(str(e))
+            logging.error("Error while pulling with rebase: %s", sanitized_error)
```

Ruff (0.14.13)

`error()` call on root logger

(LOG015)